src/components/InternalLink/index.tsx

```diff
@@ -1,39 +1,41 @@
-import Link from "next/link";
-import {useRouter} from "next/router";
-import {parseLocalStorage} from "../../utils/parseLocalStorage";
+import Link from 'next/link';
+import { useRouter } from 'next/router';
+import { parseLocalStorage } from '../../utils/parseLocalStorage';
 
-export default function InternalLink({href, children}) {
-  let filterKind = "";
-  if (href.startsWith("/cli") || href.startsWith("/console")) {
-    filterKind = "";
-  } else if (href.startsWith("/lib")) {
-    filterKind = "platform";
-  } else if (href.startsWith("/sdk")) {
-    filterKind = "platform";
-  } else if (href.startsWith("/ui")) {
-    filterKind = "framework";
-  } else if (href.startsWith("/guides")) {
-    filterKind = "platform";
-  } else if (href.startsWith("/start")) {
-    filterKind = "integration";
+export default function InternalLink({ href, children }) {
+  let filterKind = '';
+  if (href.startsWith('/cli') || href.startsWith('/console')) {
+    filterKind = '';
+  } else if (href.startsWith('/lib')) {
+    filterKind = 'platform';
+  } else if (href.startsWith('/sdk')) {
+    filterKind = 'platform';
+  } else if (href.startsWith('/ui')) {
+    filterKind = 'framework';
+  } else if (href.startsWith('/guides')) {
+    filterKind = 'platform';
+  } else if (href.startsWith('/start')) {
+    filterKind = 'integration';
   }
 
-  if (filterKind != "") {
-    const filterKeys = parseLocalStorage("filterKeys", {});
-    if (filterKind in filterKeys) {
-      const filterKey = filterKeys[filterKind];
-      if (href.includes("#")) {
-        const hrefParts = href.split("#");
-        href = `${hrefParts[0]}/q/${filterKind}/${filterKey}#${hrefParts[1]}`;
-      } else {
-        href += `/q/${filterKind}/${filterKey}`;
+  if (filterKind != '') {
+    if (!href.includes(`/q/${filterKind}/`)) {
```

Comment:
This and the corresponding `}` are the only lines actually being added everything else is just prettier updates